test/worlds/bottomless_pit.sdf

```diff
@@ -69,7 +69,7 @@
 
     <include>
       <pose>0 0 -80 0 0 1.57</pose>
-      <uri>https://fuel.gazebosim.org/1.0/arjo129/models/LRAUV-PostIonic</uri>
+      <uri>https://fuel.gazebosim.org/1.0/accurrent/models/MBARI Tethys LRAUV</uri>
```

Comment:
Can we put this model in the openrobotics account?

Reply:
Sure! I don't have the needed permission or credentials though.

Reply:
[229afb2](https://github.com/gazebosim/gz-sim/pull/2608/commits/229afb217ffe1608b244c6950bbd67c8df56028c)